src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -897,11 +897,11 @@ components:
       type: object
       required:
         - raw_hex
-        - raw_json
```

Comment:
how about `programmatic_json` to a) stay consistent with Core API b) leave room for future `natural_json`?